bin/strap

```diff
@@ -11,6 +11,7 @@ loader.eager_load
 
 opts = Slop.parse do | opts |
   opts.string '-c', '--config', required: true
+  opts.string '--cronjob', required: true
```

Comment:
can we make this optional and default to `false`?

Reply:
you can also make it cast to bool with `opts.bool '--cron', required: false, default: false`